smriprep/workflows/anatomical.py

```diff
@@ -219,12 +227,33 @@ def init_anat_preproc_wf(
                             run_without_submitting=True)
 
     # 2. Brain-extraction and INU (bias field) correction.
-    brain_extraction_wf = init_brain_extraction_wf(
-        in_template=skull_strip_template[0],
-        template_spec=skull_strip_template[1],
-        atropos_use_random_seed=not skull_strip_fixed_seed,
-        omp_nthreads=omp_nthreads,
-        normalization_quality='precise' if not debug else 'testing')
+    def _is_skull_stripped(imgs):
+        """ Checks that if T1w images are skulled stripped by interrogating
+        extreme planes for all-zeros
+        """
+        def _check_img(img):
+            data = img.get_data()
+            sidevals = data[0, :, :].sum() + data[-1, :, :].sum() + \
+                data[:, 0, :].sum() + data[:, -1, :].sum() + \
+                data[:, :, 0].sum() + data[:, :, -1].sum()
+            return sidevals > 0
```

Comment:
Should consider the case where zeroes aren't exactly zero (e.g., float data saved as int16, so scaling parameters apply and push the 0 value slightly off):

```suggestion
            return sidevals > 10
```

Does that seem reasonably big to allow accumulation across 6 planes but well under what would happen if there were neck in one of them?

Reply:
I might not be understanding the algorithm, but should the comparison be `sidevals < 10`?

My main concern would be:
  - skull-stripped images with the inferior plane cutting through the cerebellum (e.g., bad FoV prescription) would be false negatives here [although maybe you should just not use such bad scan].
  - images that have been rotated in defacing to be AC-PC aligned could potentially give false positives under certain circumstances (in-scanner denoising). Still, 10 should be sufficiently low.

Reply:
Oops, I think you're correct, it should be <10 (that is, the image is already skull stripped, and therefore should return True, and trigger n4 only workflow.

I think any heuristic is going to fail sometimes, which why I'd want to provide a CLI option to over-ride (should we allow both force skip, and force trip then?)

About #107, I'm not sure I understand. This was originally buit for a use case in which a raw BIDS dataset had skull stripped images (and were not labeled as such). This seems to be a not common enough usecase. 

I'm fine skipping the heuristic altogether though and just have a `skip-brain-extraction` CLI flag. Your call @oesteban

Reply:
> About #107, I'm not sure I understand. This was originally buit for a use case in which a raw BIDS dataset had skull stripped images (and were not labeled as such). This seems to be a not common enough usecase.

Okay, sorry I misunderstood. This PR is to add a fall-back if the user (unknowingly) is passing in skull-stripped brains. Since the user doesn't know it, we can't expect them to place those brains in the derivatives folder.

In that case, I totally understand your approach - thanks for the patience. Let's go ahead with this. I'll push for the BIDS-validator to look into this particular, so it will give you a list of skull-stripped datasets, which you can move to a derivatives folder. Until that happens (if it ever happens), having this in fMRIPrep will be helpful :).

That said, all the proposals (current and alternatives) seem fine to me - my concern came from not understanding the problem addressed.

Reply:
Yep, you got it. Again, if at any point we think the heuristic is too unreliable, we can leave it as a CLI flag. 

That said, I think the current heuristic errs on the side of skull stripping (as the extreme values have to be very low to skip skull stripping automatically). So I think a false positive (i.e. non-skull stripped T1 detected as skull stripped) is fairly rare. 

Agree the BIDS-validator should check for this. In theory these datasets are BIDS-invalid, since they are not RAW. But in practice many such datasets exist in the wild. 

I fixed the logic in the test. Let me know if you need anything else.